file-system-view.js

```diff
@@ -243,10 +244,15 @@ View.prototype.renderRow = function(fileSystemItem, showFullPath) {
 	}
 };
 
-View.prototype.renderRows = function(fileSystemItems, showFullPath) {
+View.prototype.renderRows = function(fileSystemItems, showFullPath, showHiddenFiles) {
 	var rows = [];
 	for (var i = 0; i < fileSystemItems.length; i++) {
 		var fileSystemItem = fileSystemItems[i];
+
+		if(showHiddenFiles === false && fileSystemItem.name[0] === '.' && fileSystemItem.name !== ".."){
```

Comment:
To check filesystemitem` s hidden state check `fileSystemItem.mode` See https://nodejs.org/api/fs.html#fs_class_fs_stats. It is a integer which needs to be interpreted by code. I guess this is a unix code which describes item`s state on file system.